pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -986,6 +986,14 @@ func (p *PriorityQueue) Update(logger klog.Logger, oldPod, newPod *v1.Pod) error
 	p.lock.Lock()
 	defer p.lock.Unlock()
 
+	if p.isSchedulingQueueHintEnabled {
+		// the inflight pod will be requeued using the latest version from the informer cache, which matches what the event delivers.
+		if _, ok := p.inFlightPods[newPod.UID]; ok {
+			logger.V(6).Info("The pod is currently being processed, skip update", "pod", klog.KObj(newPod))
```

Comment:
```suggestion
			logger.V(6).Info("The pod is currently being processed", "pod", klog.KObj(newPod))
```

Maybe this is enough?  "skip update" sounds wrong to user, sounds like we're not doing anything for the pod, while actually we'll try to reschedule it later.

Reply:
I agree `skip update` may give a wrong nuance, but also I agree it's worthy to put in a log.

How about:

```suggestion
			logger.V(6).Info("The queue skips enqueueing the pod to the queue for now because it's being scheduled and will be queued back if necessary", "pod", klog.KObj(newPod))
```